### ISSUE TITLE
Cannot authenticate registered and verified user via "authWithPassword".

### ISSUE THREAD

**Issue by hsedr** [pocketbase/js-sdk#130]
I am currently trying to integrate Pocketbase as the backend for my Nuxt application.
Pocketbase is already set up, running on my home server. 
In Nuxt I created an api endpoint, which is called when a user tries to login. 
The user is already registered and verfied in the 'Users' collection in Pocketbase.

```javascript
import express from "express";
const body_parser = require("body-parser");
import PocketBase from "pocketbase";
import "cross-fetch/polyfill";
import 'node-fetch';

const app = express();
app.use(body_parser.json());

app.post("/login", async (req, res) => {
  const pb = new PocketBase("http://192.168.2.51:8090");
  try {
    const { email, password } = req.body.data;
    const userData = await pb.collection('users').authWithPassword(email, password);
    if (pb.authStore.isValid) {
      console.log(pb.authStore.isValid);
      console.log(pb.authStore.token);
      console.log(pb.authStore.model.id);
    } else {
      res.statusCode = 401;
      res.send("Not authorized");
    }
  } catch (err) {
    console.log(err);
  }
});

app.get("/logout", () => {});

module.exports = app;
```
But when calling "authWithPassword" it throws an exception:

```
Not Found.                                                                                                                       15:51:10

  at new ClientResponseError (node_modules\pocketbase\dist\pocketbase.es.mjs:1:3108)
  at node_modules\pocketbase\dist\pocketbase.es.mjs:1:46319
  at processTicksAndRejections (node:internal/process/task_queues:96:5)
  at async api\auth\index.js:15:22
```

The corresponding Request Log:

![request_log](https://user-images.githubusercontent.com/98208331/212478200-5479f4c5-7972-4aa8-890a-085eeb6294ff.PNG)

On the upside i am able to fetch data from other collections. So this problem occurs only when trying to access the User auth collection.

**Comment by ganigeorgiev** [comment]
From the logs panel screenshot it seems that you are using older PocketBase version (< 0.8).

For v0.7 -> v0.8+ migration guide please follow https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#migrate-from-v07x (_the temp `upgrade` command is also available in the v0.11 binaries so you should be able to also upgrade directly from v0.7 to v0.11_).

If that's not the case, then my guess us that the issue could be somewhere with the fetch polyfill you are using (I recommend to upgrade to Node 18).

**Comment by hsedr** [comment]
Thank you! You were right. I did not pay attention to the version I download. Everything is working!
